brain/pkg/datastore/recorder/mysql/task_group_recorder.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2022 The EasyDL Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql
+
+import (
+	"github.com/intelligent-machine-learning/easydl/brain/pkg/datastore/dbbase"
+	"time"
+	"xorm.io/xorm"
+)
+
+// tableTaskGroups is the name of job metrics table
+const tableTaskGroups = "task_groups"
+
+// TaskGroup represents a task structure
+type TaskGroup struct {
```

Comment:
Do we need TaskGroup or Task? I think the resources of each pod can be stored in the column of `resource` of job_metrics.